Fixes #94 gof merged plot

pchelle opened the pull request.

pchelle committed:
Fixes #87 GOFMerged Plots

pchelle committed:
Fixes #79 Update of getConfigurationPlan

pchelle committed:
Fixes #78 Update of QualificationWorkflow

pchelle committed:
Time Profile function update

pchelle committed:
updated function of getReportFigure to account for PlotSettings input

pchelle committed:
Update runQualificationWorkflow

PlotSettings, TimeProfile and GOFMerged as treated tasks

pchelle committed:
Temporary fix of issue #88

msevestre commented on src/code/qualification/ConvertTimeUnit.m:
```diff
@@ -0,0 +1,22 @@
+function [Yout, YoutUnit] = ConvertTimeUnit(Yin, YinUnit, YoutUnit)
```
Should be named `convertTimeUnit` no?

@Yuri05 @KatrinCoboeken I am sure we have something in place for unit conversion. Please advise

msevestre commented on src/code/qualification/runQualificationWorkflow.m:
```diff
@@ -42,45 +42,75 @@ function runQualificationWorkflow(WSettings, ConfigurationPlan, TaskList, Observ
+            % pause()
             SectionPath = getSection(ConfigurationPlan.Sections, TimeProfile.SectionId);
-            saveas(gcf,fullfile(SectionPath, ['OutputTimeProfile' num2str(j)]), 'png');
+            saveas(gcf,fullfile(SectionPath, ['OutputTimeProfile' num2str(j)]), 'pdf');
```
@pchelle we want png right?

msevestre commented on src/code/qualification/runQualificationWorkflow.m:
```diff
@@ -42,45 +42,75 @@ function runQualificationWorkflow(WSettings, ConfigurationPlan, TaskList, Observ
+                    close(gcf);
+                end
+                if ~isempty(strfind(GOFMerged.PlotType, 'predictedVsObserved'))
+                    SectionPath = getSection(ConfigurationPlan.Sections, GOFMerged.SectionId);
```
Looks a lot like line 88 to 90. ALso similar to what we have somewhere else for time profile
Can we create a method that takes a SectionId, sections array and a figure and export it somewhere? 

@Yuri05 Or is this an issue because the gcf is passed as argument ? (re . copy vs ref)

msevestre commented on src/code/qualification/runQualificationWorkflow.m:
```diff
@@ -97,6 +127,7 @@ function runQualificationWorkflow(WSettings, ConfigurationPlan, TaskList, Observ
             saveas(gcf,fullfile(SectionPath, ['ComparisonTimeProfilePlots' num2str(j)]), 'png');
```
Same method for example here. Write once, reuse everywhere :)

pchelle committed:
Update plot size

chart Height and Width are not scaled by screensize
comment under the plot is currently removed

pchelle committed:
Function output axes units

Output units  of axis for converting unifying the units of the plotted data

pchelle committed:
Fixes #103 Fixes#104

Create _title.md and _content.md when creating Section folder

pchelle committed:
Fixes #103 Fixes#104 with .m file

pchelle committed:
Fixes #105 getSection path of a specific Section Id

getSection path and count all indexed files

pchelle committed:
Update with unit conversion

pchelle committed:
Add caption to plot

pchelle committed:
Add GMFE and PKTable as .md output

pchelle committed:
Update Time profile plot

msevestre commented:
@pchelle There are still some TODOs that are not implemented that were discussed previously. @Yuri05 will open separate issues for those
